core/src/components/content/content.scss

```diff
@@ -29,6 +29,7 @@
   --offset-top: 0px;
   --offset-bottom: 0px;
   --overflow: auto;
+  background: red;
```

Comment:
ooooof

Reply:
LOL I thought changes weren't building so I added that 😭